docs/tutorials/gluon/learning_rate_schedules.md

```diff
@@ -134,7 +134,7 @@ batch_size = 64
 
 # Load the training data
 train_dataset = mx.gluon.data.vision.MNIST(train=True).transform_first(transforms.ToTensor())
-train_dataloader = mx.gluon.data.DataLoader(train_dataset, batch_size, shuffle=True)
+train_dataloader = mx.gluon.data.DataLoader(train_dataset, batch_size, shuffle=True, num_workers=5)
```

Comment:
how come num_workers is hard-coded to 5 instead of using max(1,cpu_count-3)?

Reply:
Are you removing all python's in ```python? There's one on line 124.

Reply:
@vishaalkapoor having too many workers for your DataLoader slows down your processing. Ideally your workers are in `[0, cpu_count - 2 - n_gpu]`
You want to use 0 workers when your data is already in memory and pre-processed, for example an ArrayDataset that contains your data pre-transformed and ready to be fed to GPU.